nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -738,8 +738,15 @@ double RegulatedPurePursuitController::findVelocitySignChange(
     and determine it's distance from the robot. If there is no cusp in the path,
     then just determine the distance to the goal location. */
     if ( (oa_x * ab_x) + (oa_y * ab_y) < 0.0) {
-      auto x = global_plan_.poses[pose_id].pose.position.x - pose.pose.position.x;
```

Comment:
I've modified the `findvelocitySignChange` method to simply use the `transformed_plan` variable alone instead of passing the pose as well since the transformed plan is already in the robot's frame.  I've updated the corresponding tests in the test file too.  However, while the release_build passes, the release_test fails and I'm not sure I understand why.  I'm new to this so if I could get some pointers in the right direction I'd appreciate it.  Thank you.